### PR TITLE
fix: Migrations

### PR DESCRIPTION
**Pull request by elwinschmitz**

### CODE REVIEW

**Comment by RubenGeo** [comment]
@elwinschmitz Do you want someone to review this? Should I make a task for it?

**Comment by elwinschmitz** [comment]
@RubenGeo It might be useful for a solution to the “random errors in API test-runs”.

**Comment by jannisvisser** [comment]
@elwinschmitz I merged master in this (which now has the fix by Vladimir to the API-test errors). Can you comment on the further status of this?

File: services/121-service/src/notifications/message.service.spec.ts [hunk 1]
```diff
@@ -18,88 +20,197 @@ const messageJob = {
...
+      const testMessageJob = {
+        ...defaultMessageJob,
+        messageProcessType: MessageProcessType.tryWhatsapp,
+        phoneNumber: '999888777',
+        whatsappPhoneNumber: null,
+      };
+      const testNotificationText = 'WhatsApp Generic Message';
+      getNotificationTextMock = jest
+        .spyOn(messageService as any, 'getNotificationText')
+        .mockResolvedValue(testNotificationText);
```
Comment (elwinschmitz) [inline services/121-service/src/notifications/message.service.spec.ts:118]: Define a specific text for this test, so we can verify THIS text will be used.

File: services/121-service/src/notifications/message.service.spec.ts [hunk 2]
```diff
@@ -18,88 +20,197 @@ const messageJob = {
...
+        intersolveVoucherService.storeTransactionResult,
+      ).toHaveBeenCalledWith(
+        testMessageJob.customData.payment,
+        testMessageJob.customData.amount,
+        testMessageJob.registrationId,
+        1,
+        StatusEnum.waiting,
+        undefined,
+        testMessageJob.programId,
+        testMessageID,
```
Comment (elwinschmitz) [inline services/121-service/src/notifications/message.service.spec.ts:211]: Test that our real values are being used in the test...

**Pull request merged**